platform/build-faiss/build-prereqs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+export PATH=$HOME/.local/bin:$PATH
+export DEBIAN_FRONTEND=noninteractive
+
+sudo -E apt-get update && sudo -E apt-get dist-upgrade -y
```

Comment:
Chaining with `&&` is bad practice, in my very strongly held position. This style of code/scripting was reintroduced into the current development context by early versions of Docker.

The `&&` syntax in Docker was introduced because `Docker` and `overlayfs` had limits on the number of layers. I believe 38 layers, and docker imploded. It has since been passed off as an `optimization`...  Not imo.

Reply:
most of the chaining gone, resolving comment.